pkg/trace/stats/statsraw.go

```diff
@@ -20,12 +22,20 @@ const (
 	// relativeAccuracy is the value accuracy we have on the percentiles. For example, we can
 	// say that p99 is 100ms +- 1ms
 	relativeAccuracy = 0.01
+	sketchGamma = 1.015625
+	sketchOffset = 1.8761281912861705
```

Comment:
Is this offset/bias value the result of the new formula that accounts for the time unit difference?

Reply:
Yes